templates/indieauth-authorize-form.php

```diff
@@ -1,52 +1,41 @@
 <?php
-$errors = new WP_Error();
+$login_errors = new WP_Error();
 login_header(
+	/* translators: 1. Client Name */
 	sprintf( __( 'Authorize %1$s', 'indieauth' ), empty( $client_name ) ? $client_id : $client_name ),
 	'',
-	$errors
+	$login_errors
 );
 ?>
-<form method="post" action="<?php echo $url; ?>">
+<form method="post" action="<?php echo esc_url( $url ); ?>">
 	<div class="client-info">
 		<?php if ( ! empty( $client_icon ) ) { ?>
-			<img src="<?php echo $client_icon; ?>" />
+			<img src="<?php echo esc_url( $client_icon ); ?>" />
 		<?php } ?>
+		<strong> 
 		<?php
-			printf(
-				__( '<strong>%1$s wants to access your site.</strong>', 'indieauth' ), 
-				$client
-			);
+			/* translators: 1. Client */
+			printf( esc_html__( '%1$s wants to access your site.', 'indieauth' ), $client ); // phpcs:ignore
```

Comment:
This doesn't need to be phpcs:ignored. It can be escaped like this without warning:
```suggestion
			echo esc_html( sprintf( __( '%1$s wants to access your site.', 'indieauth' ), $client ) );
```